### ISSUE TITLE
'dashboard-proxy' is not a valid MicroK8s subcommand !

### ISSUE THREAD

**Issue by j75** [ubuntu/microk8s#1336]
Following the infos at https://microk8s.io/ I tried to do ` microk8s dashboard-proxy` but I get only this:
```
'dashboard-proxy' is not a valid MicroK8s subcommand.
Available subcommands are:
	add-node
	cilium
	config
	ctr
	disable
...
```

**Comment by j75** [comment]
[inspection-report-20200621_001621.tar.gz](https://github.com/ubuntu/microk8s/files/4808642/inspection-report-20200621_001621.tar.gz)

**Comment by fidesachates** [comment]
Same here.

**Comment by ktsakalozos** [comment]
`dashboard-proxy` is a recent addition. It is available on the latest track and will be available in the 1.19 track and forward. Please install MicroK8s with the `--channel` argument:
```
sudo snap install microk8s --classic --channel=latest/stable
```

**Comment by j75** [comment]
In this case please update the documentation at https://microk8s.io/ because is there where I find that 
```
 1) Install MicroK8s on Linux
sudo snap install microk8s --classic
```

**Comment by ktsakalozos** [comment]
@evilnick @joedborg, @j75 is right. With  `sudo snap install microk8s --classic` you get the 1.18/latest track that does not have the `dashboard-proxy` command. The  `dashboard-proxy` command is on the latest stable and will be available with the tracks from 1.19 and on. Should we update the docs to have the old dashboard access command or should we have the snap install to use the "latest/stable" channel?

**Comment by joedborg** [comment]
https://github.com/ubuntu/microk8s/pull/1341
